tests/integration/network-integration.cfg

```diff
@@ -1,4 +0,0 @@
-[persistent_connection]
```

Comment:
this is not required

Reply:
are you saying that this file isn't required? Because its already removed